### DISCUSSION TITLE
Wait x days to apply the update

### DISCUSSION THREAD

**Discussion by LordRofl** [containrrr/watchtower#1417] [Feature Requests]
Saw a discussion about automatic updaters and why not to use them. They say you are on the bleeding edge of every new version.
Thats true but for me its okay in this case, but i thought why not wait a given time to apply the new update. Let's say watchtower detects a new version and you can set an amount of time that needs to run down before watchtower applys the new version. Then you know okay the current version does not get any hotfixes let's say in a week and boom the update gets applied.

**Comment by simonaronsson** [comment]
That's an interesting suggestion! Would you be interested in having a stab at it? We're more than happy to support you with code reviews and what not.
